content/docs/devops-tips/prometheus-metrics.md

```diff
@@ -64,6 +64,48 @@ spec:
       honorLabels: true
 ```
 
+### TLS
+
+TLS can be enabled on the metrics endpoint for end-to-end encryption. This is achieved either using pre-signed static certificates, or using the internal dynamic certificate signing.
+
+#### Static certificates
+
+Static certificates can be provided to the cert-manager controller to use when listening on the metric endpoint. If the certificate files are changed then cert-manager will reload the certificates for zero-downtime rotation.
+
+Static certificates can be specified via the flags `--metrics-tls-cert-file` and `--metrics-tls-private-key-file` or the corresponding config file parameters `metricsTLSConfig.filesystem.certFile` and `metricsTLSConfig.filesystem.keyFile`.
+
+An example config file would be:
+
+```yaml
+apiVersion: controller.config.cert-manager.io/v1alpha1
+kind: ControllerConfiguration
+metricsTLSConfig:
+  filesystem:
+    certFile: "/path/to/cert.pem"
+    keyFile: "/path/to/key.pem"
+```
```

Comment:
I haven't tried this yet, but how would you get these files into the Pod filesystem?
Using the `.volumes` and `.mounts` Helm values?

And if so, how would the user manage those Certifcates and keys? 
It seems like the answer should be to use a cert-manager itself to create a Secret and mount that, 
but there will be a bootstrapping problem.

So maybe  we should explain that here in the docs.